editor/editor_node.cpp

```diff
@@ -1811,11 +1811,11 @@ void EditorNode::save_all_scenes() {
 	_save_all_scenes();
 }
 
-void EditorNode::save_scene_list(Vector<String> p_scene_filenames) {
+void EditorNode::save_scene_if_open(const String &p_scene_path) {
```

Comment:
This will slow down things significantly as you're now looping over all the open scenes for each scene tested

Reply:
I believe it is faster than it was before. In the previous version, for every open scene, it would do a `find()` call on the potentially very long passed vector of paths. The list of open scenes doesn't get very large (maybe 10-20 max) because they start eating memory. There probably is a way to optimize it more but this seems like an improvement to me.

Reply:
But should be a hash set instead IMO, where the check is fast